Scripts/RemediationScripts/AutoRemediation.ps1

```diff
@@ -191,6 +191,9 @@ Enter the choice (1|2)";
                     Write-Host $([Constants]::SingleDashLine)
                    $commandString = $control.InitCommand + " -SubscriptionId " +  "`'" + $SubscriptionId +  "`'" + " -FTPState " + "`'" + $FTPState +  "`'" + " -Path " + "`'" + "FailedControls\" +  $SubscriptionId + ".json" + "`'" + " -PerformPreReqCheck"+ " -AutoRemediation" + " -TimeStamp " + "`'" + $timeStampString +  "`'";              
                 }
+                elseif($control.ControlId -eq "Azure_FrontDoor_DP_Use_Secure_TLS_Version_Trial"){
```

Comment:
is there any special command string need to be prepared otherwise we can use common common string instead.

Reply:
Used as common string